utils/test.go

```diff
@@ -0,0 +1,17 @@
+package utils
+
+import "testing"
+
+// CheckNum make int assertion
+func CheckNum(num, expected int, t *testing.T) {
+	if num != expected {
+		t.Errorf("got %d, expected %d", num, expected)
+	}
+}
+
+// ChechStr make string assertion
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported function CheckStr should be of the form "CheckStr ..."